kotlin-frontend/src/main/kotlin/org/jetbrains/kotlin/gradle/frontend/webpack/GenerateWebPackConfigTask.kt

```diff
@@ -69,7 +69,7 @@ open class GenerateWebPackConfigTask : DefaultTask() {
                     .mapNotNull { it.dependencyProject }
                     .flatMap { it.tasks.filterIsInstance<Kotlin2JsCompile>() }
                     .filter { !it.name.contains("test", ignoreCase = true) }
-                    .map { project.file(it.outputFile) }
+                    .map { it.outputFile }
```

Comment:
Also see `SimpleFrontendProjectTest.kt` (in the end) for kotlin and gradle versions applied during integration test. Notice that this test takes long long for run